modules/globebrowsing/src/tileprovider/defaulttileprovider.cpp

```diff
@@ -105,24 +127,56 @@ DefaultTileProvider::DefaultTileProvider(const ghoul::Dictionary& dictionary)
     _performPreProcessing = _layerGroupID == layers::Group::ID::HeightLayers;
     _performPreProcessing = p.performPreProcessing.value_or(_performPreProcessing);
 
+    // Get the name of the layergroup to which this layer belongs
+    auto it = std::find_if(
+        layers::Groups.begin(),
+        layers::Groups.end(),
+        [id = _layerGroupID](const layers::Group& gi) {
+            return gi.id == id;
+        }
+    );
+    auto layerGroup = it != layers::Groups.end() ? it->name : std::to_string(static_cast<int>(_layerGroupID));
```

Comment:
Line length;

`auto` is okay is here since it is an iterator that would be long to write otherwise